Vanilla-JS-Projects/Intermediate/Password-Generator/Readme.md

```diff
@@ -0,0 +1,110 @@
+  <!-- ![Vue.js](https://img.shields.io/badge/vuejs-%2335495e.svg?style=for-the-badge&logo=vuedotjs&logoColor=%234FC08D) -->
+  <!-- ![MongoDB](https://img.shields.io/badge/MongoDB-%234ea94b.svg?style=for-the-badge&logo=mongodb&logoColor=white) -->
+</div>
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Description 📃</h3>
+
+
+<div align='center'>
+  <!-- <p>Add Description of the project</p> -->
+
+HTML (index.html):The HTML file defines the structure of a webpage.
+It includes a title, a link to an external CSS file for styling, and a body containing a container div.
+Inside the container div, there's an heading for the title, a paragraph for displaying the generated password, and a button triggering the password generation.
+
+CSS (style.css):The CSS file styles the HTML elements to enhance the visual presentation of the webpage.
+It centers the content on the page and gives a basic style to the button.
+
+JavaScript (script.js):The JavaScript file contains a function named generatePassword.
+This function is called when the "Generate Password" button is clicked.
+It generates a random password of a specified length (default is 12 characters) using a set of characters including uppercase and lowercase letters, digits, and special symbols.
+The generated password is then displayed in the HTML paragraph element.
+Overall Description:
+
+The webpage provides a simple user interface with a title, a display area for the generated password, and a button to trigger password generation.
+Users can click the "Generate Password" button to get a random password displayed on the webpage.
+The styling is kept minimal, and you can customize it further based on your design preferences. The JavaScript code handles the logic of generating random passwords.
+
+
+
+
+
+</div>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>How to run it? 🕹️</h3>
+
+<div align='center'>
+  <!-- <p>Add steps how to run this project</p> -->
+    <p>Open index.html in a Browser:
+
+Right-click on the index.html file.
+Select "Open with" and choose a web browser (e.g., Google Chrome, Mozilla Firefox, Safari).
+View the Webpage:
+
+The webpage should open in your selected browser.
+You will see the "Random Password Generator" title, a paragraph indicating the generated password, and a button labeled "Generate Password."
+Generate Password:
+
+Click the "Generate Password" button, and you should see a randomly generated password displayed in the paragraph below.</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Screenshots 📸</h3>
+![Alt text](Password-Generator.webp)
```

Comment:
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/b747fc4f-2eed-4b72-ac3d-25610d55592d)

@Hacxmr Screenshot is not shown in readme file.
- You have to upload the pic on comment section, image link will be obtained for you.
- As i have done for the above pic `![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/5c50ad4e-c0ae-44f6-ac3a-36164af35496)`
- Then just copy that link and paste it.